src/pkg/backup/details/groups.go

```diff
@@ -121,6 +123,14 @@ func (i GroupsInfo) Values() []string {
 			dttm.FormatToTabularDisplay(i.Message.CreatedAt),
 			lastReply,
 		}
+	case GroupsConversationPost:
+		return []string{
+			// html parsing may produce newlines, which we'll want to avoid
+			strings.ReplaceAll(i.Post.Preview, "\n", "\\n"),
```

Comment:
I think details are broken for conversations, including this branch. 

I tried `./corso backup create groups --group CorsoCITeam --data conversations` 
`  Posts                                       2s done (found 10 conversations)`

followed by `./corso backup details groups  --backup   07a2b25b-495c-4f09-9366-d4361e7b32a2`

It returns `no items match the provided selectors`.

Reply:
I'll debug it.

Reply:
@ryanfkeepers i have the details fix here - https://github.com/alcionai/corso/issues/4780 
Could you please merge it in this PR before pushing it?

Reply:
I've merged #4826 into this PR. @ryanfkeepers ok to merge this PR into main?